applications/OptimizationApplication/python_scripts/utilities/opt_line_search.py

```diff
@@ -55,7 +55,7 @@ def ComputeScaleFactor(self) -> float:
     @time_decorator()
     def ComputeStep(self) -> float:
         norm = self.ComputeScaleFactor()
-        if not math.isclose(norm, 0.0, abs_tol=1e-16):
+        if not math.isclose(norm, 0.0, abs_tol=1e-22):
```

Comment:
are you sure about those limits? This seems waaaay smaller than machine accuracy. Might be the problem is elsewhere :thinking: